### ISSUE TITLE
"KILLED" After Merging Two Big Wordlists

### ISSUE THREAD

**Issue by ghost** [k4m4/dymerge#2]
After _attempting_ to merge together two wordlists (143.9MB + 2.9GB), it ended with "KILLED".
Here is the code:
root@kali:~/dymerge# python dymerge.py '/root/Downloads/Wordlists/HashesORG251015.txt' '/root/Downloads/Wordlists/W0RD715T.txt' -s -u -o ~/Desktop/P455W0RD715T.txt
DyMerge 0.2 Nikolaos Kamarinakis (nikolaskama.me)

[+] Starting Dictionary Merge Task
[+] Reading Dictionaries
[+] Merging DictionariesKilled
root@kali:~/dymerge# 

So Basically, it broke, i'm guessing from the size of the wordlists.

**Comment by k4m4** [comment]
This occurs because you do not have enough available RAM to generate a successful merging task with dictionaries of such great size. I will probably alter some functions to modify the data stored in the RAM in the near future.

**Comment by k4m4** [comment]
Please read my latest article on DyMerge [here](https://nikolaskama.me/why-dymerge-sucks/). I believe it will help you out. Thanks.
